uaclient/api/u/pro/security/vulnerabilities/cve/v1.py

```diff
@@ -0,0 +1,383 @@
+        self.fix_status = fix_status
+        self.fix_available_from = fix_available_from
+
+
+class CVEVulnerabilityResult(DataObject):
+    fields = [
+        Field(
+            "name",
+            StringDataValue,
+            doc="The name of the CVE",
+        ),
+        Field(
+            "description",
+            StringDataValue,
+            doc="The CVE description",
+        ),
+        Field(
+            "published_at",
+            DatetimeDataValue,
+            doc="The CVE published date",
+        ),
+        Field(
+            "ubuntu_priority",
+            StringDataValue,
+            doc="The ubuntu priority for the CVE",
+        ),
+        Field(
+            "notes",
+            data_list(StringDataValue),
+            False,
+            doc="A list of notes for the CVE",
+        ),
+        Field(
+            "affected_packages",
+            data_list(CVEAffectedPackage),
+            doc="A list of affected packages for this CVE",
+        ),
+        Field(
+            "fixable",
+            StringDataValue,
+            doc="The fixable status of the CVE",
+        ),
+        Field(
+            "cvss_score",
+            FloatDataValue,
+            False,
+            doc="The CVE cvss score",
+        ),
+        Field(
+            "cvss_severity",
+            StringDataValue,
+            False,
+            doc="The CVE cvss severity",
+        ),
+    ]
+
+    def __init__(
+        self,
+        *,
+        name: str,
+        description: str,
+        published_at: datetime.datetime,
+        ubuntu_priority: str,
+        fixable: str,
+        notes: Optional[List[str]] = None,
+        affected_packages: List[CVEAffectedPackage],
+        cvss_score: Optional[float] = None,
+        cvss_severity: Optional[str] = None
+    ):
+        self.name = name
+        self.description = description
+        self.published_at = published_at
+        self.ubuntu_priority = ubuntu_priority
+        self.fixable = fixable
+        self.notes = notes
+        self.affected_packages = affected_packages
+        self.cvss_score = cvss_score
+        self.cvss_severity = cvss_severity
+
+
+class CVEVulnerabilitiesResult(DataObject, AdditionalInfo):
+    fields = [
+        Field(
+            "cves",
+            data_list(CVEVulnerabilityResult),
+            doc="A list of CVEs that affect the system",
+        ),
+        Field(
+            "vulnerability_data_published_at",
+            DatetimeDataValue,
+            doc="The date the JSON vulnerability data was published at",
+        ),
+        Field(
+            "apt_updated_at",
+            DatetimeDataValue,
+            False,
+            doc="The date of the last apt update operation in the system",
+        ),
+    ]
+
+    def __init__(
+        self,
+        *,
+        cves: List[CVEVulnerabilityResult],
+        vulnerability_data_published_at: datetime.datetime,
+        apt_updated_at: Optional[datetime.datetime] = None
```

Comment:
I think this should be None when using a manifest file

Reply:
Done